src/wmtk/operations/tri_mesh/EdgeSwapBase.cpp

```diff
@@ -103,9 +103,12 @@ bool EdgeSwapBase::execute()
 
 std::vector<Simplex> EdgeSwapBase::modified_primitives() const
 {
-    std::vector<Simplex> s;
-    s.emplace_back(simplex::Simplex::edge(m_output_tuple));
-    return s;
+    return {simplex::Simplex::edge(m_output_tuple)};
+}
+
+std::vector<Simplex> EdgeSwapBase::unmodified_primitives() const
+{
+    return {input_simplex()};
```

Comment:
weren't you the one who had a compiler complain about this return notation? i've been avoiding this for you - but i guess i can do this again ;-) 
(i think this was particularly with std::array your compiler was not happy)

Reply:
No my compiler never complained about that stuff. It was only that I didn't know that for `std::array` you should write `{{...}}` but it never was an issue.